tests/phpunit/WC_Stripe_Webhook_Handler_Test.php

```
@@ -296,6 +296,91 @@ function ( $passed_order ) use ( $order ) {
+			->method( 'process_response' )
+			->with(
+				$this->anything(),
+				$this->callback(
+					function ( $passed_order ) use ( $order ) {
+						return $passed_order instanceof WC_Order && $order->get_id() === $passed_order->get_id();
+					}
+				)
+			);
+
+		$this->mock_webhook_handler->process_deferred_webhook( 'payment_intent.succeeded', $data, $notification );
+	}
```

PHPMD (2.15.0)

[error] The method test_deferred_webhook_processes_when_order_is_in_valid_status is not named in camelCase. (undefined)

(CamelCaseMethodName)